hw_diag/tasks.py

```diff
@@ -40,14 +38,6 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
     diagnostics['PK'] = public_keys['key']
     diagnostics['AN'] = public_keys['name']
 
-    # Fetch data from miner container.
-    try:
-        diagnostics = fetch_miner_data(diagnostics)
```

Comment:
This was the only invocation of `fetch_miner_data`. Can you remove that method all together and related test code?

Reply:
It was going to be removed with light miner branch. Not sure if helium has any plans to revert on anything. But I can remove it now as well.

Reply:
There is no need for backwards compatibility at this point

Reply:
done